### DISCUSSION TITLE
Dichiarazione di adesione ad ANSC: chiarimento su indicazione relativa alla numerosità dei casi d'uso

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1094] [Supporto Webapp] [answered]
Buongiorno.
All'interno della Guida di adesione ad ANSC ([https://github.com/italia/ansc/blob/main/docs/Guida_adesione_ANSC.pdf](url)), nella sezione relativa alla dichiarazione di adesione si indica in particolare la presenza del campo:

![image](https://github.com/user-attachments/assets/82e9e15a-1f3b-49da-8827-47d587bb317b)

Come ampiamente e a più riprese condiviso, l'implementazione dei casi d'uso ANSC su gestionale locale segue una pianificazione inevitabilmente progressiva e si completerà nel corso del 2025.
Chiediamo pertanto conferma che non sia obbligatorio per il comune attivare il flag:
"Nel caso di avvenuta adozione tramite gestionale, capacità di gestione della totalità dei casi d'uso di stato civile codificati"
alla dichiarazione di adesione tramite gestionale.

Rimaniamo in attesa di vostro riscontro e naturalmente a disposizione

**Comment by asqueglia** [comment, marked as answer]
Buongiorno lasciamo la presente discussion al DTD

**Comment by migmar68** [comment]
Confermiamo la problematica dell'adozione progressiva dei casi d'uso, soprattutto quelli più rari. Tra l'altro continuano ad aggiungersene di nuovi.  Credo che la problematica sia ben presente anche al DTD, e forse ipotizzo si voleva ricevere conferma dal Comune che in qualunque caso, il nuovo atto (evento) sarebbe stato redatto in ANSC.  Questa è più una garanzia che potrebbe fornire il Sindaco, piuttosto che una sul software.
